readme.md

```diff
@@ -4,8 +4,6 @@ A simple to-do list plugin for logseq
 
 > This plugin relies solely on the Logseq Plugin API to access local data, and does not store it externally.
 
-<a href="https://www.buymeacoffee.com/yuexunjiang"><img src="https://img.buymeacoffee.com/button-api/?text=Buy me a coffee&emoji=&slug=yuexunjiang&button_colour=FFDD00&font_colour=000000&font_family=Comic&outline_colour=000000&coffee_colour=ffffff" /></a>
```

Comment:
I don't understand why you delete this.
This is my plugin, you have no right to remove the sponsor button in the README after adding these codes.